backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -450,32 +456,8 @@ public static void initializeServicesBaseOnDataBase(
         new HealthStatusManager(app.applicationContext.getBean(HealthServiceImpl.class));
     healthStatusManager.setStatus("", HealthCheckResponse.ServingStatus.SERVING);
 
-    // --------------- Start Initialize Database base on configuration --------------------------
-    LOGGER.trace("Database properties found");
-
-    switch (database.DBType) {
-      case ModelDBConstants.RELATIONAL:
-
-        // --------------- Start Initialize relational Database base on configuration
-        // ---------------
-        app.propertiesMap = propertiesMap;
-        ModelDBHibernateUtil.createOrGetSessionFactory(database);
-
-        LOGGER.trace("RDBMS configured with server");
```

Comment:
what about this check? currently we do not have other database type so you can move migration up but when second database will come up we will need this switch case.

Reply:
We'll hit that problem when we get there. The config is already checking the type and ensuring it's only relational for now.

Reply:
Ok, then it will fine for now with assuming we are supporting just RDBMS.